diff --git a/configure.ac b/configure.ac
index e511f01..0e9f549 100644
--- a/configure.ac
+++ b/configure.ac
@@ -197,17 +197,6 @@ AC_CACHE_CHECK([for VA API (GLX extensions)],
   CFLAGS="$CFLAGS $LIBVA_DEPS_CFLAGS"
   AC_TRY_COMPILE([
     #include <va/va_version.h>
-    #if VA_MAJOR_VERSION == 0 && VA_MINOR_VERSION == 30
-    # if !defined(VA_SDS_VERSION) || (VA_SDS_VERSION < $LIBVA_GLX_SDS_VERSION_0_30)
-    #  error "VA API version >= 0.30.0-sds$LIBVA_GLX_SDS_VERSION_0_30 is required"
-    # endif
-    #elif VA_MAJOR_VERSION == 0 && VA_MINOR_VERSION == 31
-    # if !defined(VA_SDS_VERSION) || (VA_SDS_VERSION < $LIBVA_GLX_SDS_VERSION_0_31)
-    #  error "VA API version >= 0.31.0-sds$LIBVA_GLX_SDS_VERSION_0_31 is required"
-    # endif
-    #else
-    # error "This version of VA API is not supported for OpenGL rendering"
-    #endif
   ], [],
   [ac_cv_libva_glx_extensions="yes"],
   [ac_cv_libva_glx_extensions="no" USE_GLX=0])
